presto-orc/src/main/java/com/facebook/presto/orc/reader/SliceDictionarySelectiveReader.java

```diff
@@ -87,16 +92,17 @@
     private final int maxCodePointCount;
     private final boolean isCharType;
 
-    private byte[] stripeDictionaryData = EMPTY_DICTIONARY_DATA;
-    private int[] stripeDictionaryOffsetVector = EMPTY_DICTIONARY_OFFSETS;
-    private byte[] currentDictionaryData = EMPTY_DICTIONARY_DATA;
+    private byte[] dictionaryData = EMPTY_DICTIONARY_DATA;
+    private int[] dictionaryOffsetVector = EMPTY_DICTIONARY_OFFSETS;
     private int[] stripeDictionaryLength = new int[0];
     private int[] rowGroupDictionaryLength = new int[0];
     private byte[] evaluationStatus;
+    private byte[] tempValue;
```

Comment:
nit: can we give it a more meaningful name?

Reply:
Renamed it to `valueWithPadding`